docs/benchmarks/media-streaming.md

```diff
@@ -51,29 +52,28 @@ Copy the **`logs`** folder from Host1 to Host2
 
     $ rsync -zarvh username@HOST1:$HOME/logs $HOME
 
-Note: Pass public key file, in case of authentication error using `-e "ssh -i /path/to/.pemfile` to the `rsync` command
+Note: Pass the public key file, in case of authentication error using `-e "ssh -i /path/to/.pemfile"` to the `rsync` command.
 
-The ${SERVER_IP} is the IP of Host1
 
 To start the client you have to first `pull` the client image and then run it. To `pull` the client image use the following command:
 
     $ docker pull cloudsuite/media-streaming:client
 
 To start the client container and connect it to the *host* network use the following command:
 
-    $ docker run -t --name=streaming_client -v $HOME/logs:/videos/logs -v $HOME/output:/output --net host cloudsuite/media-streaming:client ${SERVER_IP}
+    $ docker run -t --name=streaming_client -v $HOME/logs:/videos/logs -v $HOME/output:/output --net host cloudsuite/media-streaming:client ${SERVER_IP} ${HTTPERF_CLIENTS} ${SESSIONS} ${RATE} ${ENCRYPTION_MODE}
 
-The logs folder is mounted into the client container using `-v /path/to/logs:/videos/logs`.
+The **`logs`** folder is mounted into the client container using `-v /path/to/logs:/videos/logs`. `SERVER_IP` is the IP of Host1. `HTTPERF_CLIENTS` determines the number of distinct `httperf` processes on the client machine that will generate the load. For higher loads, it would be beneficial to increase the number of httperf clients to distribute the load among multiple processes. The default value is 4. `SESSIONS` sets the number of individual sessions that the client would request from the server to stream. Each session is assigned to a video with specific quality and the client sends HTTP requests for the chunks of the video until it is fully streamed. The **`logs`** folder describes the sessions used in the benchmark. Each session is separated from the others by blank lines. In addition, `RATE` specifies the rate at which a new session will be generated. Its unit is sessions per second. For example, if `SESSIONS` is 1000 and `RATE` is 10, it takes 100 seconds until the client generates all demanded sessions. Finally, `ENCRYPTION_MODE` has "PT" and "TLS" as valid values that determine whether the server transfers the videos in the plain text format over port 80, or it uses TLSv1.3 to encrypt the media and send them over port 443. 
```

Comment:
We need to be a bit more clear here:
- Number of individual sessions per client process, or number of sessions total?
- Forward pointer from dataset part to this part, explaining how the clients process the log.
- We should specify how the steady state works (does the client keep generating sessions or not?) and when the ramp down happens.